### PR TITLE
extmod/modbluetooth.c: Implement getter for BLE rxbuf size.

### PR DESCRIPTION
**Pull request by codefreax**

### CODE REVIEW

**Comment by dpgeorge** [comment]
Thanks, it looks good.  Maybe also update the docs, eg a one-word addition like: "rxbuf: Get/set the size in bytes ..."

Did you find a need for this, to get the current size?

**Comment by codefreax** [comment]
> Did you find a need for this, to get the current size?

While debugging an issue interactively I tried increasing rxbuf, but did not see an effect. I was wondering whether `rxbuf` was the right spelling, so I tried reading the set value. Seeing this fail increased my suspicion that something is wrong on the MP side.

I don't have a better use case in my mind; I guess it is just following "Principle of Least Astonishment". Alternatively, we could spell out in the documentation that rxbuf is write-only and mention the default value.

What do you think?

**Comment by dpgeorge** [comment]
Thanks for the explanation.

> I was wondering whether `rxbuf` was the right spelling,

If it's not a valid setting then it will raise an exception when you try to set it.  So this helps at least to know you are setting something!

> Alternatively, we could spell out in the documentation that rxbuf is write-only and mention the default value.

Some values are naturally read-only, others write-only.  Some (most?) that are writable are only ever going to be the value you write to them, and in such a case there's no real need to read it back (because your code knows what it wrote).  I guess the main reason for reading such values is to know their *default value*.  That could be documented, but it may be hard to document for some ports/BLE stacks where the default value is not know until runtime.

In this case I think it's actually worth being able to read this value, because knowing buffer size can be rather important to ensure you'll receive valid data.  Eg you might want to set it then read it to check it was at least the size you requested.  It's also quite a cheap addition, in terms of code size.

In summary, I think it's a worthy addition.

**Comment by dpgeorge** [comment]
Rebased and merged in f4726735cf9607c4d77edf83f81a11a0e2ab5c98 (with minor reformat of docs change, so it just touches one line).
